registry/registry.go

```diff
@@ -52,13 +55,11 @@ func jsonResponse(rw http.ResponseWriter, code int, msg string) {
 // NewRegistry returns Registry with configured argus client and listener
 func NewRegistry(in RegistryIn) (*Registry, error) {
 
-	argus, err := chrysom.CreateClient(in.NornRegistry.Argus, chrysom.WithLogger(in.NornRegistry.Logger))
+	argus, err := chrysom.NewBasicClient(in.NornRegistry.Argus, nil)
 	if err != nil {
 		return nil, err
 	}
-	if in.NornRegistry.Listener != nil {
-		argus.SetListener(in.NornRegistry.Listener)
-	}
+
```

Comment:
I removed `argus.SetListener(in.NornRegistry.Listener)`. The Argus/Chrysom package was changed last year to split up the Chrysom client into a BasicClient and a Listener client (https://github.com/xmidt-org/argus/issues/147). Therefore, this if statement and `SetListener()` call were no longer applicable. 

Instead, should this function also include a call to `chrysom.NewListenerClient()`? https://github.com/xmidt-org/argus/blob/7a4557764548b354c095b3df115eb57f5fbe119f/chrysom/listenerClient.go#L88

Reply:
I think the functionality moved from Argus to chrysom & we should favor chrysom APIs over argus.

Reply:
Thanks for helping me look into this!